src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -190,15 +202,21 @@ private function createFirewalls($config, ContainerBuilder $container)
 
             $configId = 'security.firewall.map.config.'.$name;
 
-            list($matcher, $listeners, $exceptionListener) = $this->createFirewall($container, $name, $firewall, $authenticationProviders, $providerIds, $configId);
+            list($matcher, $listeners, $exceptionListener) = $this->createFirewall(
+                $container,
+                $name,
+                $firewall,
+                $authenticationProviders,
+                $providerIds,
+                $configId
```

Comment:
putting this multiline is *not* fixing our coding standards. It is actually doing the opposite.

Reply:
Well, the Symfony coding standards don't have a make line length. But I don't think you can disable this part entirely in Phpstorm (and their built-in config applies a limit at 120 chars IIRC)